### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by DimaCoDeeema**

### CODE REVIEW

**Review by DimaCoDeeema — COMMENTED** [review]
files deleted

File: src/pageObject.js [hunk 1]
```diff
@@ -1,15 +1,31 @@
...
+    this.header = new Header();
+  };
+  clickOnLogo() {
+    return 'Click on the Conduit logo';
+  };
 };
 
 class ArticlePage extends PageObject {
-
+  get commentButton() {
```
Comment (drabykDi) [inline src/pageObject.js:20]: Add constructor before get commentButton() 

constructor(url, slag) {
    super(url);
    this.url = url + `${slag}`;
  };

File: src/pageObject.js [hunk 2]
```diff
@@ -1,15 +1,31 @@
...
-
+  get logo() {
+    return 'Conduit logo';
+  };
 };
 
 class PageObject {
-
+  constructor(url) {
+    this.url = url;
```
Comment (drabykDi) [inline src/pageObject.js:11]: maybe you should add URL 
like this : 
this.url = 'http://test.com';

**Review by drabykDi — COMMENTED** [review]
I left a few comments

File: src/pageObject.js (hunk 1 above)
Comment (DimaCoDeeema) [reply src/pageObject.js:20]: Tests don't run after adding. Why we need to add this constructor?

File: src/pageObject.js [hunk 3]
```diff
@@ -1,15 +1,31 @@
...
+    this.header = new Header();
+  };
+  clickOnLogo() {
+    return 'Click on the Conduit logo';
+  };
```
Comment (djtyrf312) [inline src/pageObject.js:16]:
```suggestion
  clickOnLogo() {
    return 'Click on the Conduit logo';
  };
```
task description - PageObject should contain clickOnLogo method. The method should return a string 'Click on the' + logo from the Header class

Comment (djtyrf312) [reply src/pageObject.js:16]: you don't use header class here

File: src/pageObject.js (hunk 1 above)
Comment (djtyrf312) [reply src/pageObject.js:20]: task description - ArticlePage constructor takes url and slug. Constructor should modify original url by adding slug at the end

File: src/pageObject.js [hunk 4]
```diff
@@ -1,15 +1,31 @@
...
 };
 
 class ArticlePage extends PageObject {
-
+  get commentButton() {
+    return '[Publish comment] button';
+  };
+  clickOnCommentButton() {
+    return 'Click on the [Publish comment] button';
+  };
```
Comment (djtyrf312) [inline src/pageObject.js:25]:
```suggestion
  clickOnCommentButton() {
    return 'Click on the [Publish comment] button';
  };
```
task description - ArticlePage should contain clickOnCommentButton method. The method should return a string 'Click on the' + commentButton

**Review by djtyrf312 — CHANGES_REQUESTED** [review]
check the comments

**Pull request closed without merging**